app/server.go

```diff
@@ -133,12 +135,19 @@ func InteractiveConfig(binaryName string) {
 
 	// create the keys
 	privStr, pubStr := createKeyPair()
+	ips, err := net.LookupIP(publicAddress.Host())
+	log.ErrFatal(err)
+	cert, key, err := network.NewCertKey(network.NewTLSCert(big.NewInt(1), "ch", "epfl", "dedis",
```

Comment:
@nikkolasg:
* have the possibility of setting their own value for the fields instead of "ch" and "epfl"
* or give a path to an already PEM encoded certificate + key